CONTRIBUTING.md

```diff
@@ -67,3 +67,6 @@ When contributing to this repository, please first discuss the changes you wish
 Please note, that we have a [code of conduct](./CODE_OF_CONDUCT.md), please follow it in all your interactions with the project.
 
 Once you are ready to make a change, please test it appropriately, create a pull request and describe your change in the pull request. The owners of the repository will review your changes as soon as possible.
+
+## Developer Certificate of Origin (DCO)
+Due to legal reasons, contributors will be asked to accept a DCO before they submit the first pull request to this projects, this happens in an automated fashion during the submission process. SAP uses [the standard DCO text of the Linux Foundation](https://developercertificate.org/).
```

Comment:
```suggestion
Due to legal reasons, contributors will be asked to accept a DCO before they submit the first pull request to this project. This happens in an automated fashion during the submission process. SAP uses [the standard DCO text of the Linux Foundation](https://developercertificate.org/).
```

Reply:
This is just a sentence that you have to copy from:
https://wiki.wdf.sap.corp/wiki/display/ospodocs/Guidelines+for+CONTRIBUTING.md+file